src/hivpy/sexual_behaviour.py

```diff
@@ -130,3 +144,143 @@ def update_sex_groups(self, population: pd.DataFrame):
                         jump_to_new_group = pd.Series(
                             (rands >= Pmin) & (rands < Pmax), index=rred.index)
                         population.loc[index & jump_to_new_group, "sex_behaviour"] = new_group
+
+    # risk reduction factors
+    def init_risk_factors(self, pop_data):
+        n_pop = len(pop_data)
+        self.init_rred_personal(pop_data, n_pop)
+        pop_data["rred_age"] = np.ones(n_pop)  # Placeholder to be changed each time step
+        self.update_rred_age(pop_data)
+        pop_data["rred"] = (self.new_partner_factor *
+                            pop_data["rred_personal"])
```

Comment:
Just double checking: the initial `rred` only uses these two factors?
If so, can we move the call to `self.update_rred_age` after the initial calculation, for clarity?

Reply:
This does actually require an age factor, which wasn't there originally because it was explicit in `update_sex_groups`; I will put the age factor back into the initial rred. 

I'm not really sure why rred is initialised differently (using only three factors) from the way it is updated (using far more); I think the other factors are all initialised to one anyway, but I am not sure if this is because that makes sense from a modelling point of view or because it made sense from a SAS point of view... After all if some people start the simulation with an HIV diagnosis then some of these factors ought to not be one at the start. It might be something worth asking Andrew et al. about in the next call.